lib/help/_skills/advanced berserking

```diff
@@ -0,0 +1,5 @@
+While the average berserker represents a threat (albeit a one-dimensional one), 
+a handful of Berserker tribes have truly mastered the art.  These advanced berserkers
+maintain all the reckless abandon of a typical berserker but with even further
+heightened combat prowess.  Moreover, particularly skilled berserkers can work themselves
+into a bloodlust the longer they fight.
```

Comment:
There's some grammar missing here. They can work themselves into a _more intense_ bloodlust the longer they fight, something like that?

Reply:
The grammar is correct, but I can reword to make it clearer.  The intent is to convey that bloodlust stacks can be gained in two ways - killing an enemy or, with advanced berserking, over time while fighting.